README.md

```diff
@@ -20,3 +20,10 @@ Then simply apply the migrations:
 You can now run the development server:
 
     $ pipenv run python manage.py runserver
+
+##FAQ
```

Comment:
This does not render correctly, I think `## FAQ` will work. Could you also add numbering to FAQ?